### PR TITLE
FIX: backend validation for images uploading

### PR DESCRIPTION
**Pull request by ahmgeek**

### CODE REVIEW

File: backend/app/models/image.rb [hunk 1]
```diff
@@ -3,6 +3,9 @@ class Image < ActiveRecord::Base
 
   belongs_to :user
 
+  validates_property :format, of: :cover, in: ['jpeg', 'jpg', 'bmp', 'png'], case_sensitive: false,
```
Comment (mkhatib) [inline backend/app/models/image.rb]: Change :cover to :asset

and Add gif to all the validations

File: backend/app/models/user.rb [hunk 2]
```diff
@@ -14,6 +14,9 @@ class User < ActiveRecord::Base
   has_many :recommendations, :dependent => :destroy
   has_many :comments, :dependent => :destroy
 
+  validates_property :format, of: :cover, in: ['jpeg', 'jpg', 'bmp', 'png'], case_sensitive: false,
```
Comment (mkhatib) [inline backend/app/models/user.rb]: Change :cover to :avatar

**Comment by mkhatib** [comment]
Also, please do the same in category.rb

**Comment by mkhatib** [comment]
Can you also add validation of maximum size of 5MB?

**Comment by ahmgeek** [comment]
@mkhatib what cause it to fail ? I hope it's not my code addition.

**Pull request closed without merging**